src/TLuaInterpreter.cpp

```diff
@@ -80,6 +82,9 @@ int luaopen_yajl(lua_State*);
 
 using namespace std;
 
+QTextToSpeech* speechUnit;
```

Comment:
@SlySven 

re: header definitions

Yeah that's what I thought too, then I got about 6,000 lines of errors (not an exaggeration) when it tried to redefine itself during my attemp to compile, even when wrapped in the proper `#ifdef`s. Will take another crack at it down the line, maybe I was just sleep-deprived or sommat (who am I kidding I'm always sleep deprived), but we'll see!